src/PassKit/PKEnums.cs

```diff
@@ -221,17 +223,8 @@ public enum PKPaymentMethodType : ulong
 		Debit,
 		Credit,
 		Prepaid,
-		Store
-	}
-
-	[Mac (11,0)]
-	[Watch (3,0)]
-	[iOS (9,0)]
-	[Native]
-	public enum PKPaymentSummaryItemType : ulong
-	{
-		Final,
-		Pending
```

Comment:
Did we add these in this cycle?

Reply:
sorry, confusing diffs threw me off-course here! fixing